Comp2025/src/main/java/frc/robot/subsystems/Manipulator.java

```diff
@@ -68,18 +68,21 @@
 public class Manipulator extends SubsystemBase
 {
   // Constants
-  private static final String  kSubsystemName        = "Manipulator";
+  private static final String  kSubsystemName         = "Manipulator";
 
-  private static final double  kClawSpeedAcquire     = 0.5;
-  private static final double  kClawSpeedExpel       = -0.4;
-  private static final double  kClawSpeedToShoot     = -1.0;
-  private static final double  kClawSpeedToProcessor = -0.4;
-  private static final double  kClawSpeedHold        = 0.1;
+  private static final double  kAlgaeSpeedAcquire     = 0.5;
+  private static final double  kAlgaeSpeedExpel       = -0.4;
+  private static final double  kAlgaeSpeedToShoot     = -1.0;
```

Comment:
Minor note on naming:  Any reason behind the "To" in the ToShoot and ToProcessor names? I only ask, because it's inconsistent with the others. I could say kClawSpeedToAcquire just like the others. So maybe remove the "To"?